### ISSUE TITLE
How to set  the custom learning rate.

### ISSUE THREAD

**Issue by CarryJzzZ** [TuSimple/TuSimple-DUC#24]
Hi, GrassSunFlower,
    I try to train the DUC network using the init.param, but the init.param do not  contain the aspp weights, so I want to set the different learning rate of aspp to the front layers. I just want  to know how to do this in MXNET, can you provide a eaxample. Thanks!

**Comment by GrassSunFlower** [comment]
You can implement your own [optimizer](http://mxnet.incubator.apache.org/api/python/optimization/optimization.html) to get this done.
